Makefile

```diff
@@ -93,46 +93,3 @@ exploremem:
 delve:
 	dlv test ./benchmarks -- -test.bench=.
 .PHONY: delve
-
-all: tools
-.PHONY: all
-
-tools: protobuf
-.PHONY: tools
-
-check: check_tools
-.PHONY: check
-
-check_tools:
-	@# https://stackoverflow.com/a/25668869
-	@echo "Found tools: $(foreach tool,$(notdir $(GOTOOLS)),\
-        $(if $(shell which $(tool)),$(tool),$(error "No $(tool) in PATH")))"
-.PHONY: check_tools
-
-tools-clean:
-	rm -f $(CERTSTRAP) $(PROTOBUF) $(GOX) $(GOODMAN)
-	rm -rf /usr/local/include/google/protobuf
-	rm -f /usr/local/bin/protoc
-.PHONY: tooks-clean
-
-###
-# Non Go tools
-###
-
-protoVer=0.10.0
-protoImageName=ghcr.io/cosmos/proto-builder:$(protoVer)
-
-.PHONY: lint test tools install delve exploremem explorecpu profile fullbench bench proto-gen proto-lint proto-check-breaking
-
-proto-lint:
-	@$(DOCKER_BUF) check lint --error-format=json
-.PHONY: proto-lint
-
-proto-check-breaking:
-	@$(DOCKER_BUF) check breaking --against-input $(HTTPS_GIT)#branch=master
-.PHONY: proto-check-breaking
-
-proto-gen:
```

Comment:
Do you intend to remove this or is it a mistake?

Reply:
there is no more proto in this repo.

Reply:
Ah ok 👍 I think we can remove the buf config then right?